src/models/invites-ignorer.ts

```diff
@@ -303,9 +339,9 @@ export class IgnoredInvites {
      *
      * The result is *not* validated but is guaranteed to be a non-null object.
      *
-     * @returns A non-null object.
+     * @returns A non-null object. NOT validated.
      */
-    private getIgnoreInvitesPolicies(): {[key: string]: any} {
+    private getIgnoreInvitesPolicies(): any {
```

Comment:
this needs a type - what is it?

Reply:
> this needs a type - what is it?

It's data coming from Matrix, therefore untrusted. Until I have checked it (which is what the caller functions do), I have no clue. The `any` is there specifically to mark that it's untrusted. Is there a better way to write it down?